src/createStreamingManager.ts

```diff
@@ -114,13 +116,22 @@ export async function createStreamingManager<T extends CreateStreamOptions>(
 
             if (event === StreamEvents.StreamStarted) {
                 console.log('StreamStarted', event, data);
+                analytics?.track('agent-video', {
+                    event,
+                    rtcStats: data ?? [] 
+                })
             } else if (event === StreamEvents.StreamDone) {
                 console.log('StreamDone');
             } else if (event === StreamEvents.StreamFailed) {
                 callbacks.onVideoStateChange?.(StreamingState.Stop, { event, data });
+
                 clearInterval(videoStatsInterval);
                 console.log('StreamFailed');
             } else {
+                // Better remove it, because it's too noisy, since message have too many partial messages
```

Comment:
remove
only have agent-message-recieeved when Full answer gets to sdk

Reply:
changed